cluster-provision/k8s/1.17/provision.sh

```diff
@@ -2,6 +2,9 @@
 
 set -ex
 
+# Need to have the latest kernel
```

Comment:
We would need it i think also on 1.18 ?
Do we want to make it part of the base image provisioning so we wont need to do it on
every provision ?
unless its fast.

Thanks